components/tuntap/Makefile

```diff
@@ -24,10 +24,20 @@ COMPONENT_ARCHIVE_HASH=	sha256:6ed84336b3eaf29aa227bfc0b5a6e7143431862b97113a31c
 COMPONENT_PROJECT_URL=	http://www.whiteboard.ne.jp/~admin2/tuntap/
 COMPONENT_ARCHIVE_URL= 	https://codeload.github.com/kaizawa/tuntap/legacy.tar.gz/$(COMPONENT_GIT_REV)
 
+COMPONENT_DESCRIPTION=	tuntap is a Virtual Point-to-Point network device: \
+a TAP driver for Solaris that can be used for \
+OpenVPN, OpenConnect, and vpnc. The code is based on the \
+Universal TUN/TAP driver. Some changes were made and code added \
+for supporting Ethernet tunneling feature, since the Universal \
+TUN/TAP driver for Solaris only supports IP tunneling known as TUN. \
+Note: built from github checkout commit version $(COMPONENT_GIT_REV)
```

Comment:
The phrasing was adapted from SFE as is. Still, I've now pushed a version
without this line into the PR ;)

--
Typos courtesy of GMail on my Samsung Android
09.11.2015 19:31 пользователь "Alexander Pyhalov" <notifications@github.com>
написал:

> In components/tuntap/Makefile
> <https://github.com/OpenIndiana/oi-userland/pull/1508#discussion_r44310955>
> :
>
> > @@ -24,10 +24,20 @@ COMPONENT_ARCHIVE_HASH=	sha256:6ed84336b3eaf29aa227bfc0b5a6e7143431862b97113a31c
> >  COMPONENT_PROJECT_URL=	http://www.whiteboard.ne.jp/~admin2/tuntap/
> >  COMPONENT_ARCHIVE_URL= 	https://codeload.github.com/kaizawa/tuntap/legacy.tar.gz/$(COMPONENT_GIT_REV)
> >
> > +COMPONENT_DESCRIPTION=	tuntap is a Virtual Point-to-Point network device: \
> > +a TAP driver for Solaris that can be used for \
> > +OpenVPN, OpenConnect, and vpnc. The code is based on the \
> > +Universal TUN/TAP driver. Some changes were made and code added \
> > +for supporting Ethernet tunneling feature, since the Universal \
> > +TUN/TAP driver for Solaris only supports IP tunneling known as TUN. \
> > +Note: built from github checkout commit version $(COMPONENT_GIT_REV)
>
> I'd skip "Note", as we have it in info.source-url
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenIndiana/oi-userland/pull/1508/files#r44310955>.
>